enrichers/src/test/resources/multiModuleExample/integration-tests/pom.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0"?>
```

Comment:
I don't think copy and paste the poms as a samples for the quarkus version is necessary. Also, this pom is not using anything about quarkus.